app/templates/public.hbs

```diff
@@ -15,18 +15,25 @@
     </div>
   </div>
   <div class="content {{if smallLead 'with small lead'}}">
-    <div class="ui basic segment" id="public-event-content">
-      <div class="left ui rail">
-        <div class="ui basic segment">
+    <div class="ui stackable grid " id="public-event-content">
+      <div class="three wide column">
+        {{#if device.isMobile}}
+          <button {{action "toggleMenu"}} class="ui basic icon button mobile only">
+            <i class="content icon"></i>
+          </button>
+          {{#if isMenuOpen}}
+            {{public/side-menu class='toggle menu'}}
+          {{/if}}
+        {{else}}
           {{#ui-sticky context='#public-event-content' observeChanges=true}}
             {{public/side-menu}}
           {{/ui-sticky}}
-        </div>
+        {{/if}}
       </div>
-      <div>
+      <div class="ten wide column">
         {{outlet}}
       </div>
-      <div class="right ui rail">
+      <div class="three wide column" style="height:100vh;overflow: hidden; !important">
```

Comment:
the styling is a temporary-fix to prevent content overflow in the twitter widget.